src/Governance.sol

```diff
@@ -542,17 +484,20 @@ contract Governance is Multicall, UserProxyFactory, ReentrancyGuard, Ownable, IG
         // an initiative can be registered if the registrant has more voting power (LQTY * age)
         // than the registration threshold derived from the previous epoch's total global votes
 
-        uint256 upscaledSnapshotVotes = uint256(snapshot.votes);
+        uint256 upscaledSnapshotVotes = snapshot.votes;
```

Comment:
Maybe we can get rid of the `upscale` part of the name.

Reply:
Done!